### PR TITLE
remove jquery

### PR DESCRIPTION
**Pull request by huhuanming**
Not all project include jquery, so I replace jquery  to native js.

### CODE REVIEW

**Comment by huhuanming** [comment]
@ankane

File: ahoy.js [hunk 1]
```diff
@@ -55,241 +50,219 @@
...
-      tag: $target.get(0).tagName.toLowerCase(),
-      id: $target.attr("id"),
-      "class": $target.attr("class"),
-      page: page(),
-      section: $target.closest("*[data-section]").data("section")
+      tag: target.nodeName.toLowerCase(),
+      id: target.getAttribute('id'),
+      'class': target.className,
+      page: page()  // because it is not frequently used, I think it can be deleted.
+              // section: $target.closest("*[data-section]").data("section")
```
Comment (huhuanming) [inline ahoy.js]: because it is not frequently used, I think it can be deleted.

**Comment by ankane** [comment]
Hey @huhuanming, thanks for the PR.  A few comments / questions:

- Diffs should be as small as possible. Please use the existing style (double quotes, etc).
- This'll need to be backwards compatible, so the `section` property will need to be added before it can be merged.
- What browsers have you tested this on?

**Comment by huhuanming** [comment]
(｡･∀･)ﾉﾞ 

- It's ok, i will notice  the existing style
- 😳   I ' m sorry.  I dont get what is $target.closest("*[data-section]").data("section").
- Work in IE8 and below.

**Comment by ankane** [comment]
`$target.closest("*[data-section]").data("section")` traverses up the DOM tree to find an element with a `data-section` attribute and gets it's value.

Browser compatibility is my biggest concern. We'll need a (automated) way to test this in a large range of browsers (down to IE6).

**Comment by huhuanming** [comment]
thanks for your reply.
😳  And could you give me some code sample for `$target.closest("*[data-section]").data("section")` ?  thank you... 😳

😊  For compatibility, It's okay, I will be do more compatibility test include IE6.

**Comment by ankane** [comment]
Here's an example of the section in use:

```html
<div data-section="hello-123">
  <button></button>
</div>
```

If the button is clicked, the `section` property should be set to `hello-123`.

**Comment by huhuanming** [comment]
so seat, thx 😊  .

**Pull request closed without merging**